src/main/java/net/mcreator/generator/template/TemplateHelper.java

```diff
@@ -59,4 +61,8 @@ public String obj2str(Object object) {
 		return new Gson().toJson(object);
 	}
 
+	public List<String> splitString(String specialString) {

```

Comment:
I'd barely copy the original method header, `splitString` "can be many things".
```suggestion
	public List<String> splitCommaSeparatedStringListWithEscapes(String specialInfoString) {
```